### PR TITLE
[FIX] mail: several chat window fixes

### PR DESCRIPTION
**Pull request by alexkuhn**
1. Before / After
bad-1 good-1

2. Before / After
bad-2 good-2

3. Before / After
bad-3 good-3

### CODE REVIEW

File: addons/mail/static/src/core/common/chat_hub.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class ChatHub extends Component {
         this.ui = useState(useService("ui"));
         this.bubblesHover = useHover("bubbles");
         this.moreHover = useHover(["more-button", "more-menu*"], () => {
-            this.more.isOpen = this.moreHover.isHover;
+            this.more.isOpen = this.moreHover.isHover || true;
```
Comment (seb-odoo) [inline addons/mail/static/src/core/common/chat_hub.js:27]: `|| true` makes the rest of the condition obsolete

File: addons/mail/static/tests/discuss/core/web/chat_window_new_message.test.js [hunk 2]
```diff
@@ -152,3 +152,31 @@ test('open chat from "new message" should open chat window', async () => {
     await contains(".o-mail-ChatWindow", { count: 0, text: "New message" });
     await contains(".o-mail-Thread");
 });
+
+test("Opening 'new message' twice keeps new message chat window in place", async () => {
```
Comment (seb-odoo) [inline addons/mail/static/tests/discuss/core/web/chat_window_new_message.test.js:156]: Should it stay in place? Don't we want to make it the first one?

File: addons/mail/static/src/core/common/chat_hub.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/core/common/chat_hub.js]: Oops, was there just to debug easily. Removing it

File: addons/mail/static/tests/discuss/core/web/chat_window_new_message.test.js (hunk 2 above)
Comment (alexkuhn) [reply addons/mail/static/tests/discuss/core/web/chat_window_new_message.test.js:156]: TBH both work fine, the exact positioning doesn't really matter.

The test is just a pretext for coverage of clicking on "new message" twice

**Comment by alexkuhn** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**